testsuite/features/step_definitions/common_steps.rb

```diff
@@ -531,7 +531,7 @@
 end
 
 When(/^I (deselect|select) "([^\"]*)" as a (SUSE Manager|Uyuni) product$/) do |select, product, product_version|
-  if $product == product_version
+  if $product == product_version && ENV["PROVIDER"] != "aws"
```

Comment:
Could we please set ENV["PROVIDER"] in a global variable of the kind `$provider` at the same place we instantiate `$product`? And then we use the global variable.

I know is just a matter of taste, but we did like that everywhere else.